src/js/models.js

```diff
@@ -38,15 +43,15 @@ export function getFilterObjects({
     if (!appName) {
         filterList = mergeTerminalNodes(filterTree);
     } else if (!version) {
-        filterList = mergeTerminalNodes(filterTree[appName]);
+        filterList = mergeTerminalNodes(safelyGet(filterTree, appName));
```

Comment:
Seems very similar to https://github.com/Exabyte-io/mode.js/pull/18/files#diff-7163936b7d1ced9343050062c56d0389e22e63f8a6a095577271c7091c8aba35R185 - should we isolate?

Reply:
Right now it is only in two places - mode.js/src/tree.js and here. I'm ok leaving it as is, but if you want, I'll refactor it.